### PR TITLE
added new annotation macros

### PR DESCRIPTION
**Pull request by jnegrea**
i added the new annotation macros and some examples
TBD: 
- testing, 
- add more annotation types (e.g. more fonts and accents)
- make it a package option

### CODE REVIEW

**Comment by jhuggins** [comment]
This is very cool Jeff. I wonder about replacing `f` with a non-letter character to make it easier to parse visually. e.g., `\'kA` or `\;kA`. I'm on board with adding this as an option. I'm not 100% sure I'll end up using it, but I will probably give it a try.

File: example.tex [hunk 1]
```diff
@@ -139,7 +139,50 @@ \section{annotation commands}
...
+	% \verb!\f[t]A! & $\f[t]A$ \\
+\end{tabular}
+
+And multiple font codes:
+
+\begin{tabular}{ll}
+    \verb!\f[bh]A! & $\f[bh]A$ \\
+	\verb!\f[hb]A! & $\f[hb]A$ \\
+	\verb!\f[hk]A! & $\f[hk]A$ \\
+	\verb!\f[kh]A! & $\f[ka]A$ \\
```
Comment (jhuggins) [inline example.tex:167]: Typo: should be `$\f[kh]A`$

**Comment by jnegrea** [comment]
\` is used for accents in text
\; is mskip, and overwriting this could conflict with common styles that might use it.
\/ is defined but the definition is empty, and it is awk to type that since it is two far apart right pinky moves.

can look up tex commands (on linux at least using `latexdef <command name>` on the command line (see examples below).

I think @trevorcampbell 's suggestion of \s... is the most ergonomic option that wont break anything

`(base) negrea@negrea:~$ latexdef  \;

\;:
macro:->\mskip \thickmuskip 

(base) negrea@negrea:~$ latexdef  \`

\`:
macro:->\OT1-cmd \`\OT1\` 

(base) negrea@negrea:~$ latexdef  \/

\/:
\/

(base) negrea@negrea:~$ latexdef  \|

\|:
macro:->\delimiter "026B30D 
`

**Comment by jhuggins** [comment]
The case where the non-letter character seemed the most useful was for something like `\,bA`. But it turns out non-letter macros can only be a single character:
* https://en.wikibooks.org/wiki/TeX/def
* https://tex.stackexchange.com/questions/16410/what-are-category-codes
So never mind! 

I think `\sbhA` is slightly more readable than `\fbhA` so I'm happy to go with `\s...`

**Comment by trevorcampbell** [comment]
merging this (in preparation for general cleanup work etc on the master branch) and moving @jnegrea 's comments to an issue

**Pull request merged**